sites/partners/cypress/integration/navigation.ts

```diff
@@ -22,6 +22,13 @@ describe("Navigating around the site", () => {
         cy.contains("Listings")
       })
     })
+
+    it("Forgot password does not show alert", () => {
+      cy.visit("/")
+      cy.get("aside").contains("a").click()
```

Comment:
nit: I wonder if this could be `cy.contains("Forgot password?").click()`? That seems marginally easier to reason about to me, but I also don't mind if you keep it as is :)